### PR TITLE
UMAP/PCA plots for mouse meta-analysis

### PR DESCRIPTION
**Pull request by SidhantPuntambekar**
Should solve #28

### CODE REVIEW

**Comment by SidhantPuntambekar** [comment]
@kriemo @raysinensis I have added the annotated and unannotated UMAPs for the mouse to this PR. There are also PCA plots included. Not too sure if the results are correct given the sparseness of the UMAPs but these are the initial plots.

**Comment by SidhantPuntambekar** [comment]
@kriemo Yeah I think an RMarkdown could be helpful for the meta-analysis file. I am a bit unfamiliar with the syntax of it (unless it is just a combination of R and markdown) but I could look at the antigen-tracking project that Ryan is doing to get familiar with it.

**Comment by kriemo** [comment]
Rmarkdown is a mix of markdown, r code, with a YAML header to specify output options. You just have to separate the markdown from the R (or other languages) code with the code chunk syntax. 

https://rmarkdown.rstudio.com/lesson-1.html

File: meta-analysis/Mouse-Meta-Analysis.rmd [hunk 1]
```diff
@@ -27,10 +27,35 @@ library(clustree)
 
 ## Load Mouse Atlas
 
-We will now load the mouse atlas into the R data section. This will be accomplished with the here library establishing the root of the project and then 
+The mouse atlas will now be loaded into the local R data section. This will be accomplished with the here library establishing the root of the project and then loading in the mouseAtlas.rds file into the mouseAtlas matrix object. A Seurat object will then be created in order to perform single cell analysis on the mouse atlas. 
 
-```{r pressure, echo=FALSE}
-plot(pressure)
+```r
```
Comment (kriemo) [inline meta-analysis/Mouse-Meta-Analysis.rmd:32]: The code chunk syntax is wrong, it is being recognized as markdown syntax and formatting the code as text. To have it recognized as Rmarkdown and executed you need to surround the 

"\```r"   with   "```{r}"

Comment (SidhantPuntambekar) [reply meta-analysis/Mouse-Meta-Analysis.rmd]: See commit 1e3f825

**Comment by raysinensis** [comment]
Hi Sid, a couple minor comments:
1) to avoid printing out all the warnings and verbose messages, add the following to the setup chunk:
```
knitr::opts_chunk$set(warning = FALSE)
knitr::opts_chunk$set(message = FALSE)
```
2) as you can see, the HoverLocator interactive plot from Seurat works in html, which is nice. It would be nice to set `label = FALSE` to clean up the view.
```
AnnotatedUMAP <- DimPlot(mouseMetaAnalysis, reduction = "umap", label = FALSE, pt.size = 0.5) 
```
3) Some of the figures with side-by-side panels get squished. You can set individual figure width and height, like this:
`{r hierarchy of clusters, fig.width=15}`. Personally I would do so for Preprocessing and Find Variable Feature. Alternatively, this can be set globally as well, with `knitr::opts_chunk$set(fig.width=15)`, but in this case it appears to break the interactive plot sizes.
4) For the last chunk of plotting hierarchy, I would add 1 or 2 more tiers of resolutions, like adding this as the first line of the chunk:
```
mouseMetaAnalysis <- FindClusters(mouseMetaAnalysis, resolution = 1.0)
```

File: R/musMusculus/GSE124952/GSE124952.r [hunk 2]
```diff
@@ -23,7 +23,7 @@ meta_PFC
 sum(colnames(mat_PFC) %in% meta_PFC$CellType)
 ncol(mat_PFC)
 
-source("~/Reference-Matrix-Generation/R/utils/check.r")
+source("~/Reference-Matrix-Generation/R/utils/utils.r")
```
Comment (kriemo) [inline R/musMusculus/GSE124952/GSE124952.r:26]: switch to using `here()` before merging.

Comment (SidhantPuntambekar) [reply R/musMusculus/GSE124952/GSE124952.r]: See commit 09b05f9

**Comment by SidhantPuntambekar** [comment]
@kriemo @raysinensis I will merge this PR and then fix anything related to GSE124952 on the master branch.

**Pull request merged**